tests/config/utils/create-sandbox.js

```diff
@@ -0,0 +1,14 @@
+"use strict";
```

Comment:
Are we using double quotes in this file?

Reply:
All files for tests have been copied from `prettier` and have minimal changes. because of that we are using prettier's default configuration for this folder alone.